cmake/common.cmake

```diff
@@ -162,6 +162,11 @@ set(CCF_ENDPOINTS_SOURCES
     ${CCF_DIR}/src/endpoints/endpoint_registry.cpp
     ${CCF_DIR}/src/endpoints/base_endpoint_registry.cpp
     ${CCF_DIR}/src/endpoints/common_endpoint_registry.cpp
+    ${CCF_DIR}/src/endpoints/authentication/authentication_types.cpp
```

Comment:
For cpp files, where the collision risk is low, suffixes like _auth may or may not be necessary?